ovos_persona_server/vector_stores.py

```
@@ -0,0 +1,670 @@
+        # 2. Query the vector database
+        # TODO: Implement filtering based on request.filters
+        search_results = db_embeddings.query(
+            embeddings=query_embedding,
+            top_k=request.max_num_results,
+            collection_name=vector_store_id,
+            return_metadata=True
+        )
+
+        # 3. Format the results
+        data = []
+        for key, score, metadata in search_results:
```

Ruff (0.12.2)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)